core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.ts

```diff
@@ -422,6 +450,28 @@ export class CkEditor4RteComponent implements AfterViewInit, OnChanges,
     this.ckEditorCopyContentService.bindPasteHandler(ck);
   }
 
+  disableRTEicons(): void {
+    // Add disabled cursor pointer to the icons.
+    // TODO(#12882): Remove the use of jQuery.
+    this.disableOnOfflineNames.forEach((name) => {
+      // TODO(#12882): Remove the use of jQuery.
+      $('.cke_button__oppia' + name)
+        .css('background-color', '#cccccc')
+        .css('pointer-events', 'none');
+      $('#offline-warning').css('display', '');
+    });
+  }
+  enableRTEicons(): void {
+    // TODO(#12882): Remove the use of jQuery.
+    this.disableOnOfflineNames.forEach((name) => {
+      // TODO(#12882): Remove the use of jQuery.
+      $('.cke_button__oppia' + name)
+        .css('background-color', '')
+        .css('pointer-events', '');
+      $('#offline-warning').css('display', 'none');
```

Comment:
ditto here

Reply:
As for now we can not remove the jquery for ```.cke_button_oppia``` because these classes and tags including these classes are added dynamically in the template. There is no such predefined template for ckeditor.

Reply:
You can, using elementRef with ViewChild decorators can cover the dynamically added elements too.
You can take help from this: https://github.com/ashutoshc8101/oppia/commit/2f050a9cf15a99bc00e375fc572239cc97e6f2ff

Reply:
Done.